webpack.config.js

```diff
@@ -1 +1,2 @@
 module.exports = require('skatejs-build/webpack.config');
+module.exports.module.loaders[2].query.plugins = ['transform-class-properties'];
```

Comment:
> suggestions?

Not really. Feel free to proceed — just wanted to make sure this was a conscious decision and make you feel a bit guilty for it :P